scripts/bench_summary.lean

```diff
@@ -197,7 +199,8 @@ def addBenchSummaryComment (PR : Nat) (repo : String)
   IO.FS.writeFile (tempFile ++ ".src") bench
 
   -- Extract all instruction changes whose magnitude is larger than `threshold`.
-  let threshold := s!"{10 ^ 9}"
+  let thr := 10 ^ 9
+  let threshold := s!"{thr}"
```

Comment:
optional: if I see this correctly and this is indeed just used once below (I think the second time you use `{formatDiff thr`}` instead) then you could remove the var `threshold` and rename the current `thr : Nat` into `threshold` instead.

Reply:
Thanks Jon!

My intention had bee to have a single entry point for the `Nat`-valued `threshold`, but I was modifying code that already had a single entry to the `String`-valued one, hence the awkwardness!  I revised the code following your suggestion, so now `threshold` really is the single `Nat`-valued "variable" and the `String` version is in-lined: I am happy with the change!